spec/factories/authentication_token.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :valid_authentication_token, class: 'AuthenticationToken' do
+    created_by_external_user
+    shared_secret { SecureRandom.hex(32) }
+    expire_at { 7.days.from_now }
+  end
+
+  factory :invalid_authentication_token, class: 'AuthenticationToken' do
+    created_by_external_user
+    shared_secret { SecureRandom.hex(32) }
+    expire_at { 8.days.ago }
```

Comment:
As we are modifying this attribute only, I would have expected it to be a `authentication_token` factory with and `:invalid` trait.